samples/snippets/csharp/VS_Snippets_CLR_System/System.Threading.ThreadPriority/cs/Example1.cs

```diff
@@ -29,7 +29,7 @@ static void Main()
 
 class PriorityTest
 {
-    static bool loopSwitch;
+    volatile static bool loopSwitch;

```

Comment:
```suggestion
    static volatile bool loopSwitch;
```
Think it's more common for `volatile` to be placed before the type, in the sense the type can be thought of as `volatile bool`